### PR TITLE
Update cordova build title

### PR DESCRIPTION
**Pull request by Kalovelo**
refs #1421 
Renamed the cordova-build type to cordova-build(apk).
Apk in parenthesis was chosen since cordova will probably be used for iOS too in the future so a similar naming could probably be considered suitable. Also, the build description contains the keyword "Android"  giving a straightforward explanation for new Game Developers or Users that .apk is the Android type

### CODE REVIEW

**Comment by 4ian** [comment]
Unfortunately this solution does not work because the type in GDevelop must be the same as the one sent to/returned by the server - which is `cordova-build`. What you want instead is change the UI so that these "names" are translated to something that is human friendly.

**Comment by Kalovelo** [comment]
Thanks for your answer!
A solution I thought was by checking the build type on the specific component output but I didn't consider it the best idea having that "extra" check. I will proceed by implementing it since it is probably, the best one.

**Comment by 4ian** [comment]
This is fine because "cordova-build" is the internal name. We can have a component or even a function that is available to do the "human translation" :) And then reuse it anywhere the name is displayed on screen.

File: newIDE/app/src/Export/Builds/BuildsList.js [hunk 1]
```diff
@@ -26,6 +26,19 @@ const styles = {
   },
 };
 
+const formatBuildText = buildType => {
+  switch (buildType) {
+    case 'cordova-build':
+      return 'Apk Build (Cordova)';
```
Comment (4ian) [inline newIDE/app/src/Export/Builds/BuildsList.js]: All strings in GDevelop must be translatable. Use <Trans> component

File: newIDE/app/src/Export/Builds/BuildsList.js [hunk 2]
```diff
@@ -26,6 +26,19 @@ const styles = {
   },
 };
 
+const formatBuildText = buildType => {
```
Comment (4ian) [inline newIDE/app/src/Export/Builds/BuildsList.js]: Add Flow typing to buildType

**Comment by Kalovelo** [comment]
Thank you for your comments.
Since Cordova will be used for iOS, another check will probably be needed in the future.
I pulled the latest changes from master and semaphoraci seems to be failing at piskel-editor.zip (internal server error) 🤔

File: newIDE/app/src/ObjectsList/index.js [hunk 3]
```diff
@@ -295,15 +295,12 @@ export default class ObjectsList extends React.Component<Props, State> {
...
-
-    if (getObjectWithContextName(objectWithContext) === newName) {
-      return;
-    }
-
     this.setState({
       renamedObjectWithContext: null,
     });
 
+    if (getObjectWithContextName(objectWithContext) === newName) return;
```
Comment (4ian) [inline newIDE/app/src/ObjectsList/index.js]: Make sure to merge again master in your branch to avoid this file from appearing

**Comment by 4ian** [comment]
> semaphoraci seems to be failing at piskel-editor.zip (internal server error) 🤔

Seems like a transient issue, I've relaunched it.

> Since Cordova will be used for iOS, another check will probably be needed in the future

Yep you're right :) 

> I pulled the latest changes from master

Strange that another file is showing in this PR 🤔Can you retry? Otherwise we'll go ahead like this.

File: newIDE/app/src/Export/Builds/BuildsList.js [hunk 4]
```diff
@@ -26,6 +26,21 @@ const styles = {
   },
 };
 
+const formatBuildText = (
+  buildType: 'cordova-build' | 'electron-build' | 'web-build'
+) => {
+  switch (buildType) {
+    case 'cordova-build':
+      return <Trans>Apk Build (Cordova)</Trans>;
```
Comment (4ian) [inline newIDE/app/src/Export/Builds/BuildsList.js]: Let's use the same terminology as in the export:

- `Android Build` 
- `Windows/macOS/Linux Build`
- `Web (upload online) Build`

(.apk is a "technical detail", might be an .aab later) (same for Cordova/Electron - it's true but less relevant for the user)

File: newIDE/app/src/ObjectsList/index.js (hunk 3 above)
Comment (4ian) [reply newIDE/app/src/ObjectsList/index.js]: (if this does not solve the problem, you can either ignore this or close this PR and open another - I think it's a GitHub bug)

**Comment by Kalovelo** [comment]
You were right, the second pull fixed it 🎉

**Comment by 4ian** [comment]
Looks good :) Merging this now, thanks for working on this!

**Pull request merged**